extras/docker/Dockerfile

```diff
@@ -9,8 +9,10 @@ RUN cd /src/vpp && \
 
 FROM ubuntu:xenial
 WORKDIR /
+COPY --from=build-env /src/vpp/build-root/vpp_18*.deb /src/vpp/build-root/vpp-plugins*.deb /src/vpp/build-root/vpp-dev*.deb /src/vpp/build-root/vpp-lib*.deb /src/vpp/build-root/vpp-api-python*.deb /src/vpp/build-root/vpp-api-lua*.deb /src/vpp/build-root/vpp-api-java*.deb /tmp/
```

Comment:
since you are already striping it, could you also remove the lua and java bindings?

Reply:
makes virtually no difference ... i ll commit the change anyway